libs/ledgerjs/packages/hw-app-icon/src/utils.ts

```diff
@@ -0,0 +1,58 @@
+/********************************************************************************
+ *   Ledger Node JS API for ICON
+ *   (c) 2016-2017 Ledger
+ *
+ *  Modifications (c) 2018 ICON Foundation
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ ********************************************************************************/
+
+// TODO use bip32-path library
```

Comment:
don't forget

Reply:
@lvndry  This comment is from the Icon developer. We will change it when they update it.